demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2242,6 +2257,29 @@ export class DemoMeetingApp
     }
   }
 
+  populateFilterList(
```

Comment:
nit: rename to preview specific such as `populateVideoPreviewFilterList`

Reply:
Done